tsconfig.json

```diff
@@ -21,6 +21,6 @@
     "target": "es5",
     "newLine": "lf",
     "types": [],
-    "declaration": true
+    "declaration": true,
```

Comment:
Unfortunately this is invalid JSON syntax and will break many JSON parsers (including Node's default `JSON.parse()` implementation IIRC)